src/util/map.js

```diff
@@ -146,3 +146,16 @@ export function merge<K, V>(
   }
   return result;
 }
+
+/**
+ * Push a value into the sub-array at key k, creating the array if needed.
```

Comment:
What do you mean by “sub-array”?

This doesn’t technically indicate what the initial value of the array
“if needed” is, but probably the reader can infer this.

Reply:
Rewrote it